yamale/yamale_error.py

```diff
@@ -0,0 +1,5 @@
+class YamaleError(ValueError):
+    results: []
+    def __init__(self, results):
+        self.results = results
+        self.message = '\n'.join([str(x) for x in results])
```

Comment:
Instead of setting `message` here, you'll want to make a call to the super class to set the string. Otherwise we get something like this when catching and printing out a `ValueError`:
```
Validation failed!
[<yamale.schema.validationresults.ValidationResult object at 0x106078550>]
```

Reply:
OK, I didn't understand properly how inheritance work in Python, I will correct it.